src/Assets/ugui-mvvm/BindingMode.cs

```diff
@@ -3,8 +3,8 @@
     public enum BindingMode
     {
         OneTime,
-        OneWayToView,
```

Comment:
Please add Obsolete attributes for the BindingMode enum values that were renamed as well (don't forget to make them the same value).

Reply:
Tried this, but even with:
```c#
[Obsolete("Use OneWayToTarget.")]
[EditorBrowsable(EditorBrowsableState.Never)]
[HideInInspector]
OneWayToView = OneWayToTarget,
```
The "old" enum names show up in the drop down in the inspector, which is super confusing:
![image](https://user-images.githubusercontent.com/5084643/59153235-75f77080-8a09-11e9-8f64-efe0dfaafe9a.png)
Any idea how to hide the former enum names without further customizing the editor?

Reply:
Tried this, but even with:
```c#
[Obsolete("Use OneWayToTarget.")]
[EditorBrowsable(EditorBrowsableState.Never)]
[HideInInspector]
OneWayToView = OneWayToTarget,
```
The "old" enum names show up in the drop down in the inspector, which is super confusing:
![image](https://user-images.githubusercontent.com/5084643/59153235-75f77080-8a09-11e9-8f64-efe0dfaafe9a.png)
Any idea how to hide the former enum names without further customizing the editor?

Reply:
@jbruening any other thoughts here? I'm inclined to either remove OneWayToView and OneWayToViewModel (and risk breaking an existing consumer), or put this PR on hold (☹) and separately try to further customize the inspector to better handle this.

Reply:
@AdamMitchell-ms created a custom `PropertyDrawer` for `BindingMode` to exclude obsolete enums.